fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogFragment.kt

```diff
@@ -115,7 +125,20 @@ class TranslationsDialogFragment : BottomSheetDialogFragment() {
                                             from = BrowserDirection.FromTranslationsDialogFragment,
                                         )
                                     },
-                                    onTranslateButtonClick = {},
+                                    onTranslateButtonClick = {
```

Comment:
This is just a very basic default translate interaction to get the necessary files started. (Originally planned to also populate the dropdowns here, but it felt like the PR was growing too fast.)

Split work to [Bug 1874441](https://bugzilla.mozilla.org/show_bug.cgi?id=1874441) to add the dropdown data and populate the request off of those choices.

Split work to [1874477](https://bugzilla.mozilla.org/show_bug.cgi?id=1874477) to manage the pending states that'll occur during translating.